processor/src/main/java/org/mapstruct/ap/internal/model/AbstractBaseBuilder.java

```diff
@@ -50,7 +50,27 @@ public B method(Method sourceMethod) {
         return myself;
     }
 
-    boolean isDisableSubMappingMethodsGeneration() {
+    /**
+     * Checks if MapStruct is allowed to generate an automatic sub-mapping between {@code sourceType} and @{code
```

Comment:
Well we added `disableSubMappingMethodsGeneration` to the `@Mapper` and `@MapperConfig` with #1146. We don't have it in the documentation